.github/workflows/container-test-deploy.yml

```diff
@@ -1,7 +1,11 @@
 name: Build, Test, Deploy
 on:
   push:
-    branches: [main]
+    branches:
+      - main
+    tags:
+      - 'v*'
```

Comment:
Looks like this is required by `type=semver` and the consequence is that we have to use `vX.X.X` as our tag syntax, which is fine I guess.

Reply:
Yeah, we don't necessarily need to use `type=semver` and can always adjust that usage in the future if we choose and do more custom labeling. But I think `vX.X.X` is reasonable if we're doing semver. I think that's what we do on `oscal-react-library` and follows GitHub's recommendations on how to name tags.

From https://github.com/EasyDynamics/oscal-editor-deployment/releases/new:
![image](https://user-images.githubusercontent.com/850893/156789098-a47d40db-77b3-4cb8-84e2-c876e7f65be6.png)

If we don't want to semver then we can do something a little different/smarter here over time.

Reply:
No, it's fine as is.